### PR TITLE
Adding new function to be able to give control to get the value when is exporting csv using the default function

### PR DESCRIPTION
**Pull request by iagolaguna**
## Related Issue
#1318

## Description
Added new function to  `getFieldValueCsv` to get the value only inside `defaultExportCsv`,
if `getFieldValueCsv` be `undefined`, will be used the default function `getFieldValue`.
more details can be read in the issue #1318 

I'm waiting your feedback :)

### CODE REVIEW

**Comment by iagolaguna** [comment]
it's missing the type of the new function but, I have a question, in `Options` we will add the dynamic type that extends from RowData or we will move the new function to other property that already use the dynamic type ?

**Comment by mvitkin** [comment]
If I understand correctly, this feature is meant to control how a column is exported to csv, in the case that a column needs to be exported in a different format than it is rendered in the table? I believe this prop should go into the column definition, so it can be applied to a specific column.

**Comment by iagolaguna** [comment]
Yes, you understand well, I get your point and agree with you, I'm going to make those changes now

**Comment by iagolaguna** [comment]
@mvitkin  sorry for the late I was without time to proceed with this feature, but now I'm here, so  I made some changes, now I think it's more in the way that you suggested, what are you think, the code needs other change?

File: types/index.d.ts [hunk 1]
```diff
@@ -127,6 +127,7 @@ export interface Column<RowData extends object> {
   editable?: ('always' | 'onUpdate' | 'onAdd' | 'never' | ((columnDef: Column<RowData>, rowData: RowData) => boolean));
   removable?: boolean;
   render?: (data: RowData, type: ('row' | 'group')) => any;
+  getFieldValue:(rowData: RowData, columnDef: Column<RowData>) => boolean
```
Comment (mbrn) [inline types/index.d.ts]: Actually, return type is not boolean. It is commonly string.

File: src/prop-types.js [hunk 2]
```diff
@@ -45,6 +45,7 @@ export const propTypes = {
     editable: PropTypes.oneOfType([PropTypes.func, PropTypes.oneOf(['always', 'onUpdate', 'onAdd', 'never'])]),
     removable: PropTypes.bool,
     render: PropTypes.func,
+    getFieldValue: PropTypes.func,
```
Comment (mbrn) [inline src/prop-types.js]: Can you change name of function as getExportValue?

**Comment by mbrn** [comment]
Hi @iagolaguna ,

Sorry for late response. But can you make some changes, please?

**Comment by iagolaguna** [comment]
@mbrn  thank you for looking this PR, I will make those changes and return to you

File: src/prop-types.js (hunk 2 above)
Comment (mbrn) [reply src/prop-types.js]: I think that you didn't see this comment.

**Comment by mbrn** [comment]
It seems that you didn't apply changes that i want. You can reopen this after complete changes.

**Pull request closed without merging**